client/src/app/core/core-services/active-meeting.service.ts

```diff
@@ -105,4 +113,19 @@ export class ActiveMeetingService {
             ]
         };
     }
+
+    private cleanModelSubscription(): void {
+        this.hasModelSubscriptionInitiated = false;
```

Comment:
this is somewhat implicit. First, it is more likely a `shouldModelSubscribtionBeInitialized` since you set it to false if something chages. Seconds, it is a sideeffect of this methid - move this line up to the handler of the meeting id subscription, so it is clear, when it is again free to initialize